docs/howtos/cluster-resource-override.md

```diff
@@ -0,0 +1,208 @@
+          clusterSelectorTerms:
+            - labelSelector:
+                matchLabels:
+                  env: prod
+        jsonPatchOverrides:
+          - op: remove
+            path: /rules/0/verbs/2
+```
+The `ClusterResourceOverride` object above will remove the verb "list" in the `ClusterRole` named `secret-reader` on 
+clusters with the label `env: prod`.
+
+### Cluster Selector
+To specify the clusters to which the override applies, you can use the `clusterSelector` field in the `OverrideRule` spec.
+The `clusterSelector` field supports the following fields:
+- `clusterSelectorTerms`: A list of terms that are used to select clusters.
+    * Each term in the list is used to select clusters based on the label selector.
+
+### JSON Patch Override
+To specify the changes to be applied to the selected resources, you can use the `jsonPatchOverrides` field in the `OverrideRule` spec.
+The `jsonPatchOverrides` field supports the following fields:
+- `op`: The operation to be performed. The supported operations are `add`, `remove`, and `replace`.
+    * `add`: Adds a new value to the specified path.
+    * `remove`: Removes the value at the specified path.
+    * `replace`: Replaces the value at the specified path.
+
+
+- `path`: The path to the field to be modified.
+    * Some guidelines for the path are as follows:
+        * Must start with a `/` character.
+        * Cannot be empty.
+        * Cannot contain an empty string ("///").
+        * Cannot be a TypeMeta Field ("/kind", "/apiVersion").
+        * Cannot be a Metadata Field ("/metadata/name", "/metadata/namespace"), except the fields "/metadata/annotations" and "metadata/labels".
+        * Cannot be any field in the status of the resource.
+    * Some examples of valid paths are:
+        * `/metadata/labels/new-label`
+        * `/metadata/annotations/new-annotation`
+        * `/spec/template/spec/containers/0/resources/limits/cpu`
+        * `/spec/template/spec/containers/0/resources/requests/memory`
+
+
+- `value`: The value to be set.
+    * If the `op` is `remove`, the value cannot be set.
+
+
+### Multiple Override Patches
+You may add multiple `JSONPatchOverride` to an `OverrideRule` to apply multiple changes to the selected cluster resources.
+```yaml
+apiVersion: placement.kubernetes-fleet.io/v1alpha1
+kind: ClusterResourceOverride
+metadata:
+  name: cro-1
+spec:
+  clusterResourceSelectors:
+    - group: rbac.authorization.k8s.io
+      kind: ClusterRole
+      version: v1
+      name: secret-reader
+  policy:
+    overrideRules:
+      - clusterSelector:
+          clusterSelectorTerms:
+            - labelSelector:
+                matchLabels:
+                  env: prod
+        jsonPatchOverrides:
+          - op: remove
+            path: /rules/0/verbs/2
+          - op: remove
+            path: /rules/0/verbs/1
+```
+The `ClusterResourceOverride` object above will remove the verbs "list" and "watch" in the `ClusterRole` named 

```

Comment:
It's hard for the reader to get to the conclusion that "list and watch" are removed. Provide the example of clusterRole yaml file?